src/main/java/seedu/mtracker/model/Instrument.java

```diff
@@ -77,6 +84,18 @@ public void editName(HashMap<String, String> editedParameters) {
         setName(editedParameters.get(NAME_ATTRIBUTE));
     }
 
+    public void editDoneStatus(HashMap<String, String> editedParameters) {
+        if (!editedParameters.containsKey(DONE_ATTRIBUTE)) {
+            return;
+        }
+        if (editedParameters.get(DONE_ATTRIBUTE).equals(DONE_INDICATOR)) {
+            markAsDone();
+        } else {
+            assert (editedParameters.get(DONE_ATTRIBUTE).equals(UNDONE_INDICATOR));
+            markAsNotDone();

```

Comment:
Actually can remove the else block here and in the if block just return after markAsDone()